### PR TITLE
Add new instance types r5, r5d, z1d

### PR DESCRIPTION
**Pull request by rekcah78**
https://github.com/kubernetes/kops/issues/5527
https://github.com/kubernetes/kops/issues/5451

### CODE REVIEW

File: upup/pkg/fi/cloudup/awsup/machine_types.go [hunk 1]
```diff
@@ -805,6 +805,138 @@ var MachineTypes []AWSMachineTypeInfo = []AWSMachineTypeInfo{
 		EphemeralDisks: nil,
 	},
 
+	// r5 family
+	{
+		Name:           "r5.large",
+		MemoryGB:       16,
+		ECU:            nil,
```
Comment (mikesplain) [inline upup/pkg/fi/cloudup/awsup/machine_types.go]: You'll have to include ECU's as well

Comment (rekcah78) [reply upup/pkg/fi/cloudup/awsup/machine_types.go]: fixed in last commit

**Comment by mikesplain** [comment]
I'll take a look
/assign
/unassign @andrewsykim

**Comment by mikesplain** [comment]
Thanks for the pr @rekcah78, much appreciated! This looks good to me. Do you mind squashing your commits into a single one? It makes things easier down the road.  Thanks!

**Comment by rekcah78** [comment]
@mikesplain squash done

**Pull request merged**